program/lib/password_validator.rb

```diff
@@ -3,4 +3,9 @@
 # - include at least one of these characters: !, @, $, %, &
 
 def valid?(password)
+    if (password.length >=8) && (password =~ /[!@$%&]/)
```

Comment:
Hi Simone — nice job! I'm wondering if there's a way to make this read more clearly though. For example, when I see a regular expression, I realise I'm going to have to do some detective work to figure out what this particular one will do.

You can make life easier on your colleagues (and your future self) by abstracting a regex into a variable. This means your code might look something like this:

```ruby
def valid?(password)
  special_characters_to_match = /[!@$%&]/
  if (password.length >=8) && (password =~ special_characters_to_match)
        true
    else
        false
    end    
end
```